flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -1414,8 +1414,29 @@ object AggregateUtil {
           aggregates(index) = udagg.getFunction
           accTypes(index) = udagg.accType
 
-        case unSupported: SqlAggFunction =>
```

Comment:
Since we add a dedicated case for `COLLECT`, this case should not be remain at the end of this `match`.